src/globalmon/server.py

```diff
@@ -53,18 +54,36 @@ def main():
 
     period = args.period
 
+    # Define the log file location
+    log_file = "/var/log/globalmon/globalmon.log"
```

Comment:
better than hardcoding the path here would be to allow also overriding by env variable to customize the path if needed.

look at:

 https://github.com/stackmon/apimon/blob/94702abc34ff7833f5099b2a2e4d55634c036091/scripts/entrypoint.sh#L41

https://github.com/stackmon/apimon/blob/94702abc34ff7833f5099b2a2e4d55634c036091/etc/apimon.yaml#L8

https://github.com/stackmon/apimon/blob/main/etc/logging.conf